build-scripts/build-parameters.cake

```diff
@@ -0,0 +1,312 @@
+#load "./constants.cake"
```

Comment:
General comments for this file:
* There are a lot of commented code
* Mix of tabs and spaces for indention

Reply:
I'll fix the tabs and remove any commented-out code that isn't related to future functionality. Once @ChrisMaddock provides direction on future functionality, I'll either implement it or delete it.

Reply:
Responded below - this PR is pretty huge from a reviewing perspective - would appreciate if we could keep future functionality for a separate PR for now! 😄